src/BlackLinter.php

```diff
@@ -0,0 +1,94 @@
+<?php
+/**
+ * Copyright 2019 Pinterest, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Formats Python files using Black
+ */
+final class BlackLinter extends ArcanistExternalLinter {
+  private $linelen = null;
+
+  public function getInfoName() {
+    return 'Black';
+  }
+
+  public function getInfoURI() {
+    return 'https://black.readthedocs.io/';
+  }
+
+  public function getInfoDescription() {
+    return pht('Black is an opionionated code formatter for Python');
+  }
+
+  public function getLinterName() {
+    return 'BLACK';
+  }
+
+  public function getLinterConfigurationName() {
+    return 'black';
+  }
+
+public function getLinterPriority() {
```

Comment:
```suggestion
  public function getLinterPriority() {
```

Reply:
Is this whitespace change suggestion intended?